patches/patch1.diff

```diff
@@ -0,0 +1,10026 @@
+diff --git a/data/meson.build b/data/meson.build
```

Comment:
Please add a comment section at the top of the patch saying what it is for.

Reply:
Done.